client/src/pages/Notifications/NotificationsList/NotificationsList.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { Notification } from '../../../interface/Notifications';
+import timeSince from '../RelativeTimes';
+import useStyles from './useStyles';
+import Grid from '@material-ui/core/Grid';
+import Avatar from '@material-ui/core/Avatar';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Divider from '@material-ui/core/Divider';
+import Typography from '@material-ui/core/Typography';
+
+interface Props {
+  notifications: Notification[];
+  type: 'page' | 'dropdown';
+}
+export function NotificationsList({ notifications, type }: Props): JSX.Element {
+  const classes = useStyles();
+  // TODO: Mark notification as read when clicked
+  // const [invisible, setInvisible] = React.useState(false);
+
+  // const handleBadgeVisibility = () => {
+  //   setInvisible(true);
+  // };
+  return (
+    <List component="nav" className={classes.root} aria-label="notifications">
+      {notifications.map(
+        (notification: {
```

Comment:
Maybe use the Notification interface here?

Reply:
That slipped my mind! Thanks!